tests/core/conftest.py

```diff
@@ -0,0 +1,33 @@
+import asyncio
+from typing import (
+    Generator,
+)
+
+import pytest
+
+from lahja import (
+    Endpoint,
+    EventBus,
+)
+
+
+@pytest.fixture(scope='session')
+def event_loop() -> Generator[asyncio.AbstractEventLoop, None, None]:
```

Comment:
I was pretty sure that `pytest-asyncio` already exposes an `event_loop` fixture automatically.  If there is a reason we aren't using that one or why we're overriding it can you add a comment explaining.

Reply:
:+1: Didn't know that. Works! Will update and merge.